mirrord/layer-win/src/hooks/files/mod.rs

```diff
@@ -296,6 +297,16 @@ unsafe extern "system" fn nt_create_file_hook(
             return run_original();
         }
 
+        // Some Windows APIs try to open the drive root (e.g. `C:\`) to inspect metadata. We don't
+        // support mirroring root directory handles, so let Windows handle these locally.
+        if is_nt_path_disk_root(&name) {
```

Comment:
i think this can be replaced by checking if path_to_unix_path is `/` for now

it will always *exclusively* be a disk root, unless i'm somehow mistaken, because of `is_nt_path_disk_path`